### PR TITLE
1. C++  2. new releases

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: etc/nginx/snippets/rewrites.conf [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # NOTE rewrite uses 302 redirect by default (assuming replacement does not start with a protocol)
 # current versions
-set $current_version_sdk_c_api '3.3.5';
-set $current_version_sdk_dotnet_api '3.4.5';
+set $current_version_sdk_c_api '3.3.7';
+set $current_version_sdk_cxx_api '1.0.0-dp.5';
```
Comment (leeming) [inline etc/nginx/snippets/rewrites.conf:4]: do you want non x.y.z versions being published?

**Review by osfameron — APPROVED** [review]
👍 
(not keen on the git commit subject formatting if I'm gonna niggle, a list would be better as a markdown list in the body of the commit message)

File: etc/nginx/snippets/rewrites.conf (hunk 1 above)
Comment (osfameron) [reply etc/nginx/snippets/rewrites.conf:4]: should be OK, as this is only declaring a variable, not adding the repo to the playbook

Comment (RichardSmedley) [reply etc/nginx/snippets/rewrites.conf:4]: As it stands, we're not putting docs up before GA, so the situation won't arise (this is a placeholder),
but it's not impossible that beta docs would be needed here or elsewhere.

**Pull request merged**